operate/client/src/modules/components/DiagramShell/styled.tsx

```diff
@@ -25,7 +25,7 @@ const Container = styled.div`
   display: flex;
   justify-content: center;
   align-items: start;
-
+  height: 100%;
```

Comment:
Fix (Part 2): This enables the built-in scrolling behavior of dmn-js. The benefit is that the header of the decision table and the first column are always visible (sticky) while scrolling:

![image](https://github.com/camunda/zeebe/assets/2543743/2d1774c1-1546-46e6-8ea2-74935eb6d29b)